clang/test/DebugInfo/CXX/callsite-edges.cpp

```diff
@@ -0,0 +1,112 @@
+// RUN: %clang_cc1 -triple=x86_64-linux -disable-llvm-passes -emit-llvm \
+// RUN:            -debug-info-kind=constructor -dwarf-version=5 -O1 %s \
+// RUN:            -o - | FileCheck %s -check-prefix CHECK-EDGES
+
+// The following are identified edge cases involving the method being called:
+// 1) Method is declared but not defined in current CU.
+// 2) Pure virtual method but not defined in current CU.
+// 3) Virtual method defined very deeply.
```

Comment:
"very deeply" seems a bit vague - could you be more specific about what conditions are a problem/why they might be a problem?